pkgs/test/lib/src/runner/browser/compilers/dart2js.dart

```diff
@@ -115,7 +116,7 @@ class Dart2JsSupport extends CompilerSupport with JsHtmlWrapper {
         import 'package:test/src/bootstrap/browser.dart';
         import 'package:test/src/runner/browser/dom.dart' as dom;
 
-        import '${await absoluteUri(dartPath)}' as test;
+        import '${await absoluteUri(dartPath)}' as $testLibraryImportPrefix;
```

Comment:
I would call this `testSuiteImportPrefix`, `testLibrary` sounds like an import of package:test